src/main/resources/application.yml

```diff
@@ -68,6 +77,10 @@ interventions-ui:
   locations:
     sent-referral: "/referrals/{id}/confirmation"
 
+community-api:
+  locations:
+    sent-referral: "/secure/offenders/crn/{crn}/referral/sent"
```

Comment:
🙋‍♂️ Could we inline this to the code, please?

The request URL configuration won't change between environments and has the same pattern as `interventions-ui.locations.sent-referral`, which is for a different purpose, which will be confusing

Reply:
(Also just noticed there's `communityapi.baseUrl` and `community-api.locations....` (different root key)

Reply:
hm this is my bad - i think i requested it to go into config to mirror the interventions-ui location url

Reply:
i still think it's suitable to live in config

i think by it's nature this is "configuration" (vs code)